stempy/image.cpp

```diff
@@ -295,36 +295,59 @@ std::vector<double> getContainer(const STEMImage& inImage, const int numBins)
   return container;
 }
 
-// function that computes histogram for all the STEM images
+// function that computes numHist histograms for input STEM image
 // each histogram is a vector<int>
-std::vector<int> createSTEMHistogram(const STEMImage& inImage,
-                                     const int numBins,
-                                     const std::vector<double> bins)
+std::vector<std::vector<int>> createSTEMHistogram(
+  const STEMImage& inImage, const int numHist, const int numBins,
+  const std::vector<double> bins)
```

Comment:
We probably should have done this in the original PR, but I think this should be `const std::vector<double>&` instead of `const std::vector<double>` to prevent an extra copy being made.

Reply:
Sure, I have made the change, thanks!